assets/styles/_helpers.scss

```diff
@@ -15,3 +15,11 @@
     display: none;
   }
 }
+
+.text-black {
```

Comment:
Why is that needed? Bootstrap should have text-white and text-black by default...